### PR TITLE
Redo blog post meta stuff

### LINKED ISSUE #1199: Blog reading time

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] Add simple reading time (close #1199)
- [x] Split styling into more files for better maintenance
- [x] Add author pic to meta
- [x] Fix blog tags in blog post header
- [x] Remove (or edit) blog post footer 
- [x] Fix tags on mobile because some 🇩🇪  tags are crazy long

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
screen shot 2018-07-11 at 12 32 35

**Comment by RonaldDijkstra** [comment]
C'mon hound...

**Comment by RonaldDijkstra** [comment]
Deployed to Staging

https://staging.defacto.nl/blog/fort-und-weiterbildung-fur-medizinische-fachangestellte-gute-perspektiven/

File: source/blog/_blog-article-tags.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="blog-tags">
+  <% link_to "/blog/tags/" do %>
+    <span class="icon-tags"></span>
+  <% end %>
+  <% article.tags.each_with_index do |tag, index| %>
+    <%= link_to capitalize(tag), tag_path(tag, "blog"), class: "tag" %>
+    <% unless index == article.tags.size - 1 %>
```
Comment (sn3p) [inline source/blog/_blog-article-tags.erb]: Think this is a bit easier to read:

```
<% if index < article.tags.size - 1 %>
```

File: source/stylesheets/components/_case-sidebar.scss [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  .case-logo {
+    display: block;
+    margin: 0 auto;
+    max-width: 300px;
+    width: 100%;
+  }
+
+  div {
```
Comment (sn3p) [inline source/stylesheets/components/_case-sidebar.scss]: This feels a bit generic, maybe add a class?

File: source/stylesheets/components/_case-sidebar.scss [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    .case-quote-position {
+      color: $testimonial-gray;
+      font-size: $base-font-size * 0.8;
+      padding-top: 0.5em;
+    }
+  }
+
+  ul {
```
Comment (sn3p) [inline source/stylesheets/components/_case-sidebar.scss]: Same as above? And maybe nest the `li` section so it belongs to this particular list.

File: source/blog/_blog-article-meta.erb [hunk 4]
```diff
@@ -4,16 +4,16 @@ author_name = author ? blog_author_name(author) : article.data.author
...
 
 <div class="blog-meta">
-  <span class="author-name"><%= author_name %></span>
+  <span class="author">
+    <% if author && src = team_avatar_url(author) %>
+      <span class="author-image">
+        <%= image_tag src, alt: author_name, class: "avatar" %>
+      </span>
+    <% end %>
+    <span class="author-name"><%= author_name %></span>
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/blog/_blog-article-meta.erb]: No...

File: source/blog/_blog-article-tags.erb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<div class="blog-tags">
+  <% link_to "/blog/tags/" do %>
+    <span class="icon-tags"></span>
```
[the remark this answers is not in the document]
Comment (RonaldDijkstra) [reply source/blog/_blog-article-tags.erb]: `<%= link_to "", "/blog/tags/", class: "icon-tags" %>`

**Comment by RonaldDijkstra** [comment]
```
- ./build/blog/nieuwe-privacywet/index.html
  *  External link https://www.mitopics.nl/2015/07/14/forse-verandering-in-privacywetgeving-per-1-januari-2016/ failed: 500 No error
```

Haha 💩

**Comment by RonaldDijkstra** [comment]
🍏

**Pull request merged**